docs/mkdocs.yml

```diff
@@ -146,6 +147,6 @@ markdown_extensions:
 extra:
   generator: false
 
-dev_addr: 127.0.0.1:8081
+dev_addr: 127.0.0.1:10002
```

Comment:
Was this intentional? Will just require updating the development/docs.md instructions

Reply:
No, it wasn't intentional. I just a had a few port conflicts in my local environment. At some point, we have to align the ports between the projects. I'll revert his change.